Qausim/react-weather-app
# Implement Search Component

#### What does this PR do?
Implements the search component

#### Description of Task to be completed?
Write out the search component

#### How should this be manually tested?
Clone this repo, cd into it and run `npm start` then visit `localhost:8080` in a browser, the search bar should be on the page

#### Any background context you want to provide?
User should be able to enter search data for a location
